assets/components/cards/render/CardRow.jsx

```diff
@@ -1,26 +1,56 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import MoreNewsButton from './MoreNewsButton';
-
+import {connect} from 'react-redux';
+import {userSelector} from '../../../user-profile/selectors';
+import server from 'server';
+import {isUserAdmin} from '../../../users/utils';
 
 class CardRow extends React.Component {
     constructor(props) {
         super(props);
+        this.state = {
+            user: {}
+        };
     }
 
     componentDidMount() {
         if (this.props.isActive) {
             this.cardElem.scrollIntoView({behavior: 'instant', block: 'end', inline: 'nearest'});
         }
+
+        this.fetchUserData();
+    }
+
+    fetchUserData() {
```

Comment:
We can avoid this, as we should already be populating the user's products etc from the server on initial GET of the home page.
https://github.com/superdesk/newsroom-core/blob/develop/newsroom/wire/views.py#L133

If the required data is not there on page load, you can add it from the server and make sure it is added to the client/redux state